app/config/routing.yml

```diff
@@ -49,3 +49,12 @@ facebook_login:
 
 google_login:
     path: /connect-login/check-google
+
+sylius_sitemap:
+    path: /sitemap.{_format}
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.sitemap:showAction
+        _format: xml
+    requirements:
+        _format: xml|html
```

Comment:
According to [this](https://github.com/Sylius/Sylius/pull/3642/files#diff-f4ce62db82f2a8bc220f1209fca4c8deR51) we don't support `html`